extensions/copymoveto/src/destination-form.tsx

```diff
@@ -100,6 +114,7 @@ export default function DestinationForm(props: EditDestinationProps) {
         allowMultipleSelection={false}
         canChooseDirectories={true}
         canChooseFiles={false}
+        showHiddenFiles={preferences.showHiddenFolders}
```

Comment:
Here, use the `showHiddenFolders` preference. However, this preference is not available in package.json.

Reply:
Yes, the most important part is msising